README.md

```diff
@@ -13,31 +13,34 @@ Currently let's you easily flash notifications to the session. It also supports
     - [Displaying Notifications](#displaying-notifications)
 
 ## Installation
-
-You can install the package for your Laravel 4 project through Composer.
-
-Require the package in your `composer.json`.
-
-```
-"andheiberg/notify": "2.*"
-```
-
-Run composer to install or update the package.
+You can install the package for your Laravel project through Composer.
 
 ```bash
-$ composer update
+# Laravel 4x
+composer require andheiberg/notify:1.*
+
+# Laravel 5x
+composer require andheiberg/notify:2.*
 ```
 
 Register the service provider in `app/config/app.php`.
 
 ```php
+# Laravel 4x
 'Andheiberg\Notify\NotifyServiceProvider',
+
+# Laravel 5x
+Andheiberg\Notify\NotifyServiceProvider::class,
```

Comment:
This is not a laravel 5 feature, but granted this came in fashion by Laravel 5. Could I ask you to remove the "Laravel 4x" (the 4 lines above)?